### DISCUSSION TITLE
Keyboard backlight does not turn off after sleep

### DISCUSSION THREAD

**Discussion by werty94** [seerge/g-helper#2945] [Q&A] [answered]
Hi @seerge, 
I'm using G-Helper 0.181 on my Rog Strix G513RC (2022) model and have the following issue: after resuming from sleep sometimes the keyboard backlight does not turn off ( I have set it to turn off after 60 seconds of inactivity). I only open the lid, the laptop resumes from sleep and the keyboard's backlight stays on until I move the mouse/touchpad and then turn off after 60 seconds. I'm attaching a log from G-Helper. If you need anything else just let me know. Thank you for your great app.
[log.txt](https://github.com/user-attachments/files/16540412/log.txt)

**Comment by seerge** [comment, marked as answer]
@werty94 keyboard is turned off when windows reports that it's being inactive for more than X seconds. If it doesn't go off after waking up from sleep - it simply means that windows doesn't report that to the app at all. If moving mouse "wakes up" inactivity timer - then move a mouse or press a button every time you activate your laptop.
